imap/carddav_db.h

```diff
@@ -142,6 +142,11 @@ int carddav_getemails(struct carddav_db *carddavdb,
                       const char *vcard_uid, unsigned kind,
                       strarray_t *members, strarray_t **group_uids);
 
+/* Looks up a group by uid, optionally filtered by addressbook mailbox,
+   and returns a list of its member uids. */
```

Comment:
Add here that the strarray pointed to by `group_uids` is allocated within the function. Or alternatively do not allocate it in the function and require the caller to pre-allocate it. The latter would allow the caller to pass a stack-allocated strarray as argument.